src/app/sessions/session.module.ts

```diff
@@ -53,6 +54,7 @@ export const COMPONENTS = [
       FormsModule,
       HearingPartModule,
       CoreModule,
+      MatSortModule,
```

Comment:
We have AngularMaterialModule where we store all related stuff. It's suggested by their developers. Please move it there.

Reply:
done